server/sonar-web/src/main/js/apps/projectActivity/components/forms/AddGraphMetric.js

```diff
@@ -60,6 +60,7 @@ export default class AddGraphMetric extends React.PureComponent {
         if (
           metric.hidden ||
           isDiffMetric(metric.key) ||
+          metric.type === 'DATA' ||
```

Comment:
it might be a good idea to also filter out 'DISTRIB' metrics, the same it's done here:
https://github.com/SonarSource/sonarqube/blob/93044147ce1178f75054ed27eb520c3ec8fb30dc/server/sonar-web/src/main/js/apps/component-measures/components/App.js#L106-L108

Reply:
What are `DISTRIB` metrics ?

Reply:
For example, function or class complexity distribution. You can find them here :) https://next.sonarqube.com/sonarqube/api/metrics/search?ps=500